### PR TITLE
Mark dry-run as implemented

### PR DESCRIPTION
**Pull request by apelisse**
cc @julianvmodesto @palnabarun 

Fixes #576

### CODE REVIEW

File: keps/sig-api-machinery/0015-dry-run.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ approvers:
 editor: apelisse
 creation-date: 2018-06-21
 last-updated: 2018-06-21
```
Comment (palnabarun) [inline keps/sig-api-machinery/0015-dry-run.md]: @apelisse Can you please update the `last-updated` date too?

Comment (apelisse) [reply keps/sig-api-machinery/0015-dry-run.md]: done

Comment (palnabarun) [reply keps/sig-api-machinery/0015-dry-run.md]: Thank you :)

**Comment by palnabarun** [comment]
/lgtm

(on behalf of 1.18 Enhancements Team)

**Comment by palnabarun** [comment]
/assign @lavalamp

(for approval)

**Pull request merged**